src/shared/components/StyledButton/styledButton.scss

```diff
@@ -24,7 +24,7 @@
   height: auto;
   margin-top: 4px;
 
-  @include screen-desktop {
-    width: 24px;
-  }
+  // @include screen-desktop {
+  //   width: 24px;
+  // }
```

Comment:
I'd remove the commented code, you're not gonna need it anymore

Reply:
Thx, i missed it